diff --git a/js/pdfkit.node.mjs b/js/pdfkit.node.mjs
index e852d25eed77cbd9054fd75df94843b87769a611..7b3ff4f7de3909aa470189fc46ba04016546173e 100644
--- a/js/pdfkit.node.mjs
+++ b/js/pdfkit.node.mjs
@@ -3649,9 +3649,10 @@ const getStandardFont = name => {
     return font;
   }
   const data = font();
-  validateFontData(data, name);
-  standardFonts.set(name, data);
-  return data;
+  const unwrapped = data != null && data.default ? data.default : data;
+  validateFontData(unwrapped, name);
+  standardFonts.set(name, unwrapped);
+  return unwrapped;
 };
 
 class PDFFontFactory {
diff --git a/package.json b/package.json
index 5d4dd8c9856fc999ba7a6c2f1fbd6a7f962b2bac..b6107b822335f976a1f63d7b1041494546e3174f 100644
--- a/package.json
+++ b/package.json
@@ -155,7 +155,7 @@
       "default": "./lib/stream/browser.js"
     },
     "#standard-fonts/*": {
-      "require": "./js/standard-fonts/*.cjs",
+      "require": "./js/standard-fonts/*.mjs",
       "default": "./js/standard-fonts/*.mjs"
     }
   },
